src/Models/WebAuthnCredential.php

```diff
@@ -98,7 +98,7 @@ class WebAuthnCredential extends Model
     protected $visible = ['id', 'origin', 'alias', 'aaguid', 'attestation_format', 'disabled_at', 'is_enabled'];
 
     /**
-     * @return \Illuminate\Database\Eloquent\Relations\MorphTo&\Laragear\WebAuthn\Contracts\WebAuthnAuthenticatable
+     * @return \Illuminate\Database\Eloquent\Relations\MorphTo
```

Comment:
Some argument as above. I assume what really has been meant is

```suggestion
     * @return \Illuminate\Database\Eloquent\Relations\MorphTo<\Laragear\WebAuthn\Models\WebAuthnCredential>
```